roles/openshift_health_checker/openshift_checks/ovs_version.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=missing-docstring
+import json
+
+from openshift_checks import OpenShiftCheck, OpenShiftCheckException, get_var
+
+
+class OVSVersion(OpenShiftCheck):
```

Comment:
Wouldn't it be easier to implement if this was part of the existing `package_version` check?